development/build/index.js

```diff
@@ -230,6 +232,12 @@ testDev: Create an unoptimized, live-reloading build for debugging e2e tests.`,
             'Whether to include SES lockdown files in the extension bundle. Setting this to `false` can be useful during development if you want to handle lockdown errors later.',
           type: 'boolean',
         })
+        .option('snow', {
+          default: true,
+          description:
+            'Whether to include Snow files in the extension bundle. Setting this to `false` can be useful during development if you want to handle Snow errors later.',
+          type: 'boolean',
+        })
```

Comment:
Should we make false the default OR update the yarn start command to specify snow=false ... we currently don't have lavamoat enabled on dev mode.

Reply:
is this ok?
20fc66b99c502aa162a5497bcac4f3bd80315005

Reply:
Yup !